theano/tensor/tests/test_opt.py

```diff
@@ -1163,7 +1163,8 @@ def my_init(shp, dtype='float64', num=0):
                 fail4.append((id, out_dtype, out.dtype))
 
         failed = len(fail1 + fail2 + fail3 + fail4)
-        print("Executed", len(cases), "cases", "failed", failed)
+        if failed > 0:
```

Comment:
There is an identical `if` statement below, which raises an exception if failed > 0. I guess this `print` is not needed anymore ?

Reply:
right.